### ISSUE TITLE
More I/O event loop errors

### ISSUE THREAD

**Issue by gregoryduckworth** [fwestenberg/reolink#38]
Thanks for the fix in https://github.com/fwestenberg/reolink/issues/37, however still getting the error I/O event loop with latest version of master `2826103`. 
```
Detected I/O inside the event loop. This is causing stability issues. 
Please report issue to the custom component author for reolink_dev doing I/O at 
custom_components/reolink_dev/ReolinkPyPi/camera.py, line 282: response = requests.post(self._url, data=json.dumps(body), params=param, timeout=10)
```

**Comment by silfa718** [comment]
Updated this morning and still seeing this issue as well.

**Comment by robmack21c** [comment]
I've just installed first time today on latest HA and I'm seeing this too.  I also see the below, (line number may be off as I've been debugging the code).  This error is a direct result of the same fix.  I'm still doing further testing as I do have the install semi working (I can see the stream).

Logger: homeassistant.core
Source: custom_components/reolink_dev/camera.py:288
First occurred: 7:58:08 PM (6 occurrences)
Last logged: 8:00:43 PM

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/reolink_dev/camera.py", line 288, in update_status
    await self.hass.async_add_executor_job(self._reolinkSession.status())
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
TypeError: 'NoneType' object is not callable

**Comment by fwestenberg** [comment]
I'm really sorry for the errors. I just noticed them in my log today. I spend some time to change the API from requests to aiohttp (#40). Now it's full async and without errors. I tried to test as many as possible, but please let me know if anything occurs.

**Comment by gregoryduckworth** [comment]
Installed the latest update and things are looking good! Thanks for all your work on this @fwestenberg

**Comment by silfa718** [comment]
Everythings looks good here.

**Comment by gregoryduckworth** [comment]
Happy for this to be closed as I've not had any issues since #40

**Comment by fwestenberg** [comment]
Thanks!
